doc-content/enterprise-only/openshift/externaldb-build-proc.adoc

```diff
@@ -11,61 +11,79 @@ Complete the steps in this build procedure to provide drivers for any of the fol
 * Oracle Database
 * Sybase
 
-For the supported versions of the database servers, see https://access.redhat.com/articles/3405381#TestedConfigurations70[Red Hat Process Automation Manager 7 Supported Configurations].
+For the supported versions of the database servers, see https://access.redhat.com/articles/3405381[Red Hat Process Automation Manager 7 Supported Configurations].
 
-The build procedure creates a custom image that extends the existing {KIE_SERVER} image. The procedure pushes this custom image into a new image stream in the `openshift` namespace with the same version tag as the original image.
+The build procedure creates a custom extension image that extends the existing {KIE_SERVER} image. You must import this custom extension image into your OpenShift environment and then reference it in the `EXTENSION_IMAGE` parameter.
 
 .Prerequisites
-* You are logged in to your project in the OpenShift environment using the `oc` command as a user with the `cluster-admin` role.
-* For IBM DB2, Oracle Database, or Sybase, you downloaded the JDBC driver from the database server vendor.
+* You are logged in to your OpenShift environment using the `oc` command. Your OpenShift user must have the `registry-editor` role.
+* For Oracle Database or Sybase, you downloaded the JDBC driver from the database server vendor.
+* You have installed the following required software:
+** Docker
+** Cekit version 3
+** The following libraries and extensions for Cekit:
+*** `odcs-client`, provided by the `python3-odcs-client` or similar package
+*** `docker`, provided by the `python3-docker` or similar package
+*** `docker-squash`, provided by the `python3-docker-squash` or similar package
+*** `behave`, provided by the `python3-behave` or similar package
+*** `s2i`, provided by the `source-to-image` or similar package
```

Comment:
For each of these, I suggest "by the __ package or similar package" for clarity (assuming each are packages). IBM isn't a fan of referring to files, folders, packages, objects, etc. without saying what it is.